docs/advanced/pcidevices.md

```diff
@@ -22,6 +22,8 @@ To use the PCI devices feature, users need to enable the `pcidevices-controller`
 
 ![](/img/v1.2/vm-import-controller/EnableAddon.png)
 
+Once addon is deployed successfully, it can take a few minutes for the controller to scan and PCIDevice crd's to be available.
```

Comment:
```suggestion
Once the `pcidevices-controller` addon is deployed successfully, it can take a few minutes for it to scan and the PCIDevice CRDs to become available.
```

Reply:
Use `pcidevices-controller` to refer to the addon to stay consistent with the Harvester UI as well as the rest of this document, e.g., line 21